### PR TITLE
Implement kpod rm

### PR DESCRIPTION
**Pull request by umohnani8**
Kpod rm removes a container from the system

Signed-off-by: Ryan Cole <rcyoalne@gmail.com>
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>
Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
Taking over from https://github.com/kubernetes-incubator/cri-o/pull/783

File: cmd/kpod/rm.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	"github.com/urfave/cli"
+)
+
+var (
+	rmFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Force removal of a running container.  The default is false",
+		},
+		cli.BoolFlag{
```
Comment (mheon) [inline cmd/kpod/rm.go]: Doesn't look like this flag is implemented. Should either remove it or make it throw an error on being set.

File: cmd/kpod/rm.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			Name:   "link, l",
+			Usage:  "Remove the specified link and not the underlying container.  The default is false",
+			Hidden: true,
+		},
+		cli.BoolFlag{
```
Comment (mheon) [inline cmd/kpod/rm.go]: Same here - flag not implemented

File: docs/kpod-rm.1.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## SYNOPSIS
+**kpod** **rm** [*options* [...]] container
+
+## DESCRIPTION
+Kpod rm will remove one or more containers from the host.  The container name or ID can be used.  This does not remove images.  Running containers will not be removed without the -f option
+
+## OPTIONS
+
+**--force, f**
```
Comment (mheon) [inline docs/kpod-rm.1.md]: If we're keeping the --volume and --link flags, they should be listed in the manpage

File: cmd/kpod/rm.go (hunk 1 above)
Comment (umohnani8) [reply cmd/kpod/rm.go]: the force flag has been implemented. It is used to force remove running and created containers.

File: docs/kpod-rm.1.md (hunk 3 above)
Comment (umohnani8) [reply docs/kpod-rm.1.md]: But, the --volume and --link flags are hidden. The user wouldn't know about its existence unless they look into the code. I don't think it should be added to the manpage yet.

Comment (rhatdan) [reply docs/kpod-rm.1.md]: I agree, lets remove the flags, I am not sure they make any sense with what we have done so far,  If kpod run adds support for builtin volumes(Not sure I want it, then we could revisit removal of them).  I have no idea what the --link flag is doing.  @runcom Any idea?

```
man docker rm
...
       -l, --link=true|false
          Remove the specified link and not the underlying container. The
       default is false.
```

Comment (umohnani8) [reply docs/kpod-rm.1.md]: Fixed.

**Comment by mheon** [comment]
LGTM pending tests

**Comment by mheon** [comment]
Test for kpod rename is failing, might be a flake

**Comment by umohnani8** [comment]
Travis just timed out...

**Comment by rhatdan** [comment]
[retest]

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**